core/management/commands/seed.py

```diff
@@ -253,13 +254,15 @@ def create_programs(output=True):
         for service in services:
             s = Service()
             s.name = service
+            s.slug = f"{p.name}-{s.name}"
+
             s.available = random_bool()
             s.program = p
             s.full_clean()
             s.save()
 
             if output:
-                print("Created {}: '{}'".format(p.name, s.name))
+                print("Created {}: '{}'".format(p.slug, s.slug))
```

Comment:
I might be missing something but I don't think `p` has a slug attribute

Reply:
ah dang you're right, i don't know how that slipped back in.